transaction/core/src/tx.rs

```diff
@@ -245,7 +245,8 @@ pub struct TxIn {
 pub struct TxOut {
     /// The amount being sent.
     #[prost(message, required, tag = "1")]
-    pub amount: Amount,
+    #[digestible(name = "amount")]
+    pub masked_amount: MaskedAmount,
```

Comment:
This is the biggest change in the PR. I added digestible support for renaming the field also